src/scriv/literals.py

```diff
@@ -4,6 +4,7 @@
 
 import ast
 import os.path
+import configparser
```

Comment:
Alphabetize imports:
```suggestion
import configparser
import os.path
```